_data/social.yml

```
@@ -177,9 +177,10 @@ websites:
 
   - name: OkCupid
     url: https://www.okcupid.com/
-    twitter: okcupid
-    facebook: okcupid
     img: okcupid.png
+    tfa:
+      - sms
+    doc: https://help.okcupid.com/article/202-sms-verification
```

Another Help Scout page... 🙄 
```suggestion
    doc: https://help.okcupid.com/article/202--
```

---

I've updated as suggested...

Note: The link on `https://help.okcupid.com` is as per my original commit.